framework/wazuh/rbac/auth_context.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, auth_context: Union[dict, str] = None, role: Union[list, orm.
         if role is None:
             # All system's roles
             with orm.RolesManager() as rm:
-                roles_list = map(orm.Roles.to_dict, rm.get_roles())
+                roles_list = list(map(orm.Roles.to_dict, rm.get_roles()))
```

Comment:
Why this change is needed?

Reply:
This change is the solution found by @fdalmaup [here](https://github.com/wazuh/wazuh/pull/20527/files) to the warning messages produced by SQLAlchemy. It was helpful to include this fix to avoid long warning messages in the console log when I was debugging the API execution.

It can be discarded and merged in the future or maintained.